### ISSUE TITLE
xp in groups severely reduced

### ISSUE THREAD

**Issue by babno** [SunwellWoW/Sunwell-TBC-Bugtracker#2026]
Decription: I get 72 xp for killing a mob alone, but only 12 if I'm in a group of 2.

How it works: 

How it should work: The penalties shouldn't be nearly that bad

Source (you should point out proofs of your report, please give us some source):

**Comment by babno** [comment]
They were the same as me (both 2). Should have mentioned that I suppose.

**Comment by Mikov1605** [comment]
Questing in a group reduces the rates, in a 2 man group by 75%.

**Comment by Mikov1605** [comment]
http://wowwiki.wikia.com/wiki/Formulas:Mob_XP#Group_Experience
This how it SHOULD be

**Comment by akrom23** [comment]
I got always 50% of normal exp.

**Comment by Mikov1605** [comment]
Yeah, same. So i wonder why he reported. Maybe its only on some mobs?

**Comment by Wojtyas** [comment]
Fixed. Was modified by .blizzlike command from party members.
